docs: add misssing networkpolicy docs (#1434)
kyma-project/istio: docs/user/tutorials/01-55-send-requests-using-egress-and-mtls.md

    ```
    {"authority":"{YOUR_DOMAIN}":"outbound|443||{YOUR_DOMAIN}",[...]}
    ```

> ## Additional security by implementing Network Policies

Reviewer: ```suggestion
## Enhance Security by Implementing NetworkPolicies
```

> Istio by default cannot securely enforce that egress traffic goes through the egress gateway. It only enables the flow
> through sidecar proxies.
> Kubernetes Network Policies can be used to restrict namespace traffic, so it only goes through the istio egress gateway.
> Network policies are the Kubernetes way to enforce traffic rules in the namespace.

Reviewer: ```suggestion
By default, Istio cannot securely enforce that egress traffic is routed through the Istio egress gateway. It only enables the flow
through sidecar proxies.
However, you can use Kubernetes NetworkPolicies to restrict namespace traffic, so it only passes through the Istio egress gateway.
NetworkPolicies are the Kubernetes method for enforcing traffic rules within a namespace.
```

> > [!NOTE] Support for network policies depends on the kubernetes CNI plugin used in the cluster.
> > Make sure to check the documentation of the CNI plugin you are using.

Reviewer: ```suggestion
> [!NOTE]
> Support for NetworkPolicies depends on the Kubernetes CNI plugin used in the cluster. Check the documentation of the CNI plugin you are using.
```

> > [!WARN] In Gardener-based clusters the Network Policy restricting DNS traffic may notwork as expected.
> > It is due to local DNS service used in discovery working outside the CNI. You may need to define the `ipBlocks` with
> > the IP CIDR of the `kube-dns` service to the NetworkPolicy to allow proper DNS resolution.

Reviewer: ```suggestion
> [!WARN] 
> In Gardener-based clusters, the NetworkPolicy restricting DNS traffic may not work as expected.
> This is due to the local DNS service used in discovery working outside the CNI. You may need to define the `ipBlocks` with
> the IP CIDR of the `kube-dns` service to the NetworkPolicy to allow proper DNS resolution.
```

Reviewer: Is it also relevant for managed Kyma?

> 

Reviewer: ```suggestion

Use the `kubeconfig` of the cluster in which you've created the Istio egress gateway.

```

> 1. In the `$NAMESPACE` namespace, create a NetworkPolicy that allows only egress traffic to the istio egress gateway,

Reviewer: ```suggestion
1. In the `$NAMESPACE` namespace, create a NetworkPolicy that allows only egress traffic to the Istio egress gateway,
```
